### PR TITLE
Feat/tp2022 t16 119 document navigation sections

### PR DESCRIPTION
**Pull request by Serginko**
If the word doc has outlines (table of contents) it will be displayed  with following functionalities:
- tree structure of outlines that can be expanded 
- redirect to the specific page when the outline is clicked

### CODE REVIEW

File: src/lib/components/document/Document.tsx [hunk 1]
```diff
@@ -25,23 +26,83 @@ interface IDocumentProps {
...
     pdfjs.getDocument({ data }).promise.then((doc) => {
+      // https://medium.com/@csofiamsousa/creating-a-table-of-contents-with-pdf-js-4a4316472fff
+      // https://mozilla.github.io/pdf.js/api/draft/module-pdfjsLib-PDFDocumentProxy.html#getDestination
+
+      doc.getOutline().then(async (outline) => {
+        if (outline == null || !outline) {
+          return
+        }
+
+        // for (let i = 0; i < outline.length; i++) {
```
Comment (rastislav-balcercik) [inline src/lib/components/document/Document.tsx:70]: Remove console.logs please

File: src/lib/components/outline/Outline.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    <>
+      {outline && outline.length > 0 && (
+        <div
+          className={`fixed top-4 right-4 z-50 ${
+            isDropdownShown ? 'h-96 overflow-y-scroll' : ''
+          }`}
+        >
+          <div className="bg-white p-2 rounded-lg shadow-md">
+            <div className="flex justify-between items-center">
+              <div className="text-lg font-semibold">Table of Contents</div>
```
Comment (rastislav-balcercik) [inline src/lib/components/outline/Outline.tsx:100]: Use language mutation for this static string

**Review by rastislav-balcercik — CHANGES_REQUESTED** [review]

File: src/lib/components/document/Document.tsx (hunk 1 above)
Comment (Serginko) [reply src/lib/components/document/Document.tsx]: done in latest commit

File: src/lib/components/outline/Outline.tsx (hunk 2 above)
Comment (Serginko) [reply src/lib/components/outline/Outline.tsx]: done in latest commit. Used useTranslation() hook  and  translation text added in EN.SK json files.

**Review by robko363 — APPROVED** [review]

**Pull request merged**